sites/target.py

```diff
@@ -169,13 +169,18 @@ def submit_order(self):
         did_submit = False
 
         self.status_signal.emit(create_msg("Submitting Order", "normal"))
+        url = self.browser.current_url
         while not did_submit:
             try:
-                if len(self.browser.find_elements_by_id('creditCardInput-cvv')) > 0:
-                    self.browser.find_element_by_id('creditCardInput-cvv').send_keys(self.profile["card_cvv"])
-                self.browser.find_element_by_xpath('//button[@data-test= "placeOrderButton"]').click()
-                self.status_signal.emit(create_msg("Order Placed", "success"))
-                send_webhook("OP", "Target", self.profile["profile_name"], self.task_id, self.product_image)
-                did_submit = True
+                cvv_field = self.browser.find_element_by_id('creditCardInput-cvv')
```

Comment:
If the 'creditCardInput-cvv' element does not exist on the web page, the program won't get past this point of the program. That's why I utilized the "if len(self.browser.find_elements_by_id('creditCardInput-cvv')) > 0:" methodology.